src/Plugins/ComplexCore/src/ComplexCore/Filters/Algorithms/FillBadData.hpp

```diff
@@ -0,0 +1,54 @@
+
+#pragma once
+
+#include "ComplexCore/ComplexCore_export.hpp"
+
+#include "complex/DataStructure/DataPath.hpp"
+#include "complex/DataStructure/DataStructure.hpp"
+#include "complex/Filter/IFilter.hpp"
+
+#include <vector>
+
+namespace complex
+{
+
+struct COMPLEXCORE_EXPORT FillBadDataInputValues
+{
+  int32 minAllowedDefectSizeValue;
+  bool storeAsNewPhase;
+  DataPath featureIdsArrayPath;
+  DataPath cellPhasesArrayPath;
+  std::vector<DataPath> ignoredDataArrayPaths;
+  DataPath inputImageGeometry;
+  //  DataPath cellDataGroupPath;
```

Comment:
Is there a reason this commented out?
- If so document why in a comment
- If not remove it